packages/react/src/primitives/Pagination/PaginationItem.tsx

```diff
@@ -48,7 +48,10 @@ export const PaginationItem: React.FC<PaginationItemProps> = ({
             </Flex>
           ) : (
             <Button
-              className={ComponentClassNames.PaginationItemButton}
+              className={
+                (ComponentClassNames.PaginationItemButton,
+                `${ComponentClassNames.PaginationItemButton}--variation-link`)
```

Comment:
Wouldn't the Button primitive add this from the variation prop?

Reply:
I like it, it removes a lot of clutter.  We would need modifiers that don't share values though specifically the default state would need a new name.